go.mod

```diff
@@ -11,73 +11,73 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.2
-	golang.org/x/crypto v0.5.0 // indirect
-	google.golang.org/grpc v1.53.0 // indirect
+	github.com/stretchr/testify v1.8.4
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.11.3
-	k8s.io/api v0.26.2
-	k8s.io/apiextensions-apiserver v0.26.0
-	k8s.io/apimachinery v0.26.2
-	k8s.io/cli-runtime v0.26.0
-	k8s.io/client-go v0.26.2
+	helm.sh/helm/v3 v3.13.3
+	k8s.io/api v0.29.0
+	k8s.io/apiextensions-apiserver v0.29.0
+	k8s.io/apimachinery v0.29.0
+	k8s.io/cli-runtime v0.29.0
+	k8s.io/client-go v0.29.0
```

Comment:
You'd need to stick with v0.28.x for helm 3.13.3
https://github.com/helm/helm/blob/c8b948945e52abba22ff885446a1486cb5fd3474/go.mod#L41-L44